tools/create-app.bash

```diff
@@ -3,25 +3,29 @@
 # Create a script that given an output directory builds the current application
 # and generates an .aab file in the output dir.
 
-# Params: $1: output directory
+### Parameters ###
+
+OUTPUT_DIRECTORY = $1
+ASSETS = $2
+SPLASH_BACKGROUND_COLOR = $3
 
 ### Assets ###
 
-# Place logo in the assets file
-full_path=$(realpath $0)
-dir_path=$(dirname $full_path)
-mv ./logo-hippie.png $dir_path/../app/assets/images/
+# Place logo in the assets directory
+FULL_PATH=$(realpath $0)
+DIR_PATH=$(dirname $FULL_PATH)
+mv $ASSETS/logo.png $DIR_PATH/../app/assets/images/
 
 # Generating splash screen
```

Comment:
This is kinda fussy but, if we start our comments in present simple (e.g.: place logo in the assets directory) we could do so with the consequent comments. I'd change the splash screen and app icon comment to something like: `Generate the splash screen` and `Generate the app icon`. What do you think?